homeassistant/components/rainforest_eagle/sensor.py

```diff
@@ -46,6 +47,9 @@
         vol.Required(CONF_IP_ADDRESS): cv.string,
         vol.Required(CONF_CLOUD_ID): cv.string,
         vol.Required(CONF_INSTALL_CODE): cv.string,
+        vol.Optional(CONF_MONITORED_CONDITIONS, default=list(SENSORS)): vol.All(
+            cv.ensure_list, [vol.In(list(SENSORS))]
```

Comment:
You do not need the `list()` here
```suggestion
            cv.ensure_list, [vol.In(SENSORS)]
```